documents/customer.md

```diff
@@ -145,6 +145,244 @@ client.customer.fetch(customerId)
 
 -------------------------------------------------------------------------------------------------------
 
+### Add Bank Account of Customer
+
+```py
+customerId = "cust_N5mywh91sXB69O"
+
+client.customer.addBankAccount(customerId, {
+    "ifsc_code" : "UTIB0000194",
+    "account_number"         :"916010082985661",
+    "beneficiary_name"      : "Pratheek",
+    "beneficiary_address1"  : "address 1",
+    "beneficiary_address2"  : "address 2",
+    "beneficiary_address3"  : "address 3",
+    "beneficiary_address4"  : "address 4",
+    "beneficiary_email"     : "random@email.com",
+    "beneficiary_mobile"    : "8762489310",
+    "beneficiary_city"      :"Bangalore",
+    "beneficiary_state"     : "KA",
+    "beneficiary_country"   : "IN"
+})
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| customerId*  | string      | Unique identifier of the customer.  |
+| account_number | string      | The name of the beneficiary associated with the bank account.  |
```

Comment:
is `account_number` name of the beneficiary?

Reply:
made correction in `account_number`